src/webportal/src/app/job/job-view/fabric/JobList/Table.jsx

```diff
@@ -303,14 +305,18 @@ export default function Table() {
   } else {
     return (
       <div>
-        <ShimmeredDetailsList
-          items={filteredJobsInfo.data}
-          setKey='key'
-          columns={columns}
-          enableShimmer={isNil(filteredJobsInfo)}
-          shimmerLines={pagination.itemsPerPage}
-          selection={selection}
-        />
+        {loading ? (
+          <SpinnerLoading />
```

Comment:
maybe loading is better move outside, and the table component only focuses on rendering items

Reply:
Done.